docs/en/app-server/grpc.md

```diff
@@ -138,6 +138,14 @@ grpc:
     destroy_timeout: 60s
 ```
 
+### Emitted metrics
```

Comment:
Small clarification, `gRPC` plugin also provides standard worker metrics: https://github.com/roadrunner-server/sdk/blob/master/metrics/metrics.go, could you please add them as well? 
- Total memory for all workers
- Per worker state
- Per worker memory
- Total number of workers
- Ready workers
- Working workers
- Invalid workers

Reply:
I thought that are common metrics for plugins with workers and I should be stick only to these related with GRPC server in documentation. No problem I will update it tomorrow.

Reply:
Updated